server/@types/express/index.d.ts

```diff
@@ -16,8 +16,8 @@ declare module 'express-session' {
     returnTo: string
     nowInMinutes: number
     requestLinkForm: RequestLinkForm
-    createBarcodeAuthToken: string
-    validCreateBarcodeAuthToken: boolean
+    slmToken: string
+    validSlmToken: boolean
```

Comment:
creatBarcodeAuthToken is being used for things other than creating barcodes (e.g. saving contacts - so renmaed this to the send legal mail token i.e. the token generated by send legal mail, not HMPPS Auth)